src/main/java/seedu/address/model/procedure/Date.java

```diff
@@ -58,6 +58,11 @@ public static boolean isValidDate(String test) {
     }
 
 
+    public int compareTo(LocalDate otherDate) {
+        return validDate.compareTo(otherDate);
+    }
```

Comment:
nice abstraction of method

Reply:
Thanks for the comment! Updated in the latest version.